main.go

```diff
@@ -307,7 +307,7 @@ func validateParams() providerConfig {
 	var ecrEnabled bool
 
 	awsAccountID = os.Getenv("awsaccount")
-	if len(awsAccountID) == 0 {
+	if len(awsAccountID) == 0 || awsAccountID == "changeme" {
```

Comment:
Oh you did merge to the same! I missed that! I wonder if there's a simple way to "walk" the user through setting it up. Like when you enable the add-on from minikube, it would prompt for the right info and create the secret / configmap behind the scenes.